mods/ts/rules/nod-support.yaml

```diff
@@ -270,6 +271,19 @@ NASAM:
 NASTLH:
 	Inherits: ^Building
 	Inherits@IDISABLED: ^DisabledOverlay
+	HitShape:
+		UseOccupiedCellsOffsets: false
```

Comment:
If the main shape is smaller than the occupied area, some targetable positions would otherwise be located outside (or at least too close to the border of) the shape.